src/audio/rtnr/CMakeLists.txt

```diff
@@ -2,7 +2,8 @@
 
 add_local_sources(sof rtnr.c)
 
-sof_add_static_library(SOF_RTK_MA_API rtklib/tgl/libSOF_RTK_MA_API.a)
-sof_add_static_library(Suite_rename rtklib/tgl/libSuite_rename.a)
-sof_add_static_library(Net rtklib/tgl/libNet.a)
-sof_add_static_library(Preset rtklib/tgl/libPreset.a)
+if(CONFIG_TIGERLAKE)
```

Comment:
this is going to be not pretty very quickly. Can you just put your libs in the root of the `third_party_libraries` folder in the root?

Reply:
The libraries could be built by different core configs so I think still need place them under different folders for each platform.